src/utils/validationUtil.ts

```diff
@@ -42,6 +43,12 @@ export class ValidationUtil {
       payload.vehicleType,
       payload.testStatus,
     );
+
+    // TODO COMMENTED OUT UNTIL FEATURE TEAMS COMPLETE IVA DEFECT WORK
```

Comment:
Do we have any mechanism for configuration and/or feature flagging that we could use instead of commenting out code?

Reply:
Not that I know of